src/Chapter4.hs

```diff
@@ -259,7 +259,7 @@ name.
 
 > QUESTION: Can you understand why the following implementation of the
   Functor instance for Maybe doesn't compile?
-
+Pattern Matching will fail since the case for Nothing isn't considered
```

Comment:
That's not entirely correct. The function pattern matches on the `Just` constructor and then on variable `x`. Since the only constructor left for the `Maybe` type is `Nothing`, the variable `x` will be `Nothing`.